docs/HowToUsePeripheral.md

```diff
@@ -307,3 +307,82 @@ Initialization is done either as part of _up_intitialize()_ or as part of _board
 Please refer to **[i2c.h](../os/include/tinyara/i2c.h)** to know more about the i2c functions and declarations.  
 
 ## I2S
+
+Each board, shall;  
+
```

Comment:
How about write a summary at top of topic?
```
There are three parts to port I2S, A, B and C.
### A
### B
### C (e.g. it can be i2s_initialize)
```